docs/cli.rst

```diff
@@ -168,6 +237,13 @@ Example: Install an Install FrameWork (IFW):
     aqt tool linux tools_ifw 4.0 qt.tools.ifw.40
 
 
+Example: List the variants of IFW available:
+
+.. code-block:: bash
+
+    aqt list tools linux desktop --tool tools_ifw   # prints 'qt.tools.ifw.40'
```

Comment:
It is better to add more examples.
There are so many users who want to know  an instant solution for their needs.